storage/memory/memory_test.go

```diff
@@ -439,6 +497,45 @@ func TestSubjects(t *testing.T) {
 	}
 }
 
+func TestSubjectsOffset(t *testing.T) {
+	ts, ctx := getTestOffsetTriples(t), context.Background()
+	g, _ := NewStore().NewGraph(ctx, "test")
```

Comment:
Same here.
Can you check the error NewGraph returns?

Reply:
Done